.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish to PyPI
+
+on:
+  release:
```

Comment:
Is this triggered when we tag a commit for release?

Reply:
No, this is triggered when we create a release on GitHub. I thought this would be good to do since we can also add release notes along with a tag when creating a release on GitHub.